package.json

```diff
@@ -11,6 +11,9 @@
 		"type": "git",
 		"url": "https://github.com/dojo/loader.git"
 	},
+	"dependencies": {
+		"dojo-core": "dojo/core"
```

Comment:
@kfranqueiro I mentioned [previously](https://github.com/dojo/loader/pull/16#issuecomment-112583621) I could use some guidance to more fully test the plugin API, because I totally agree that this PR is purely focused on the text plugin.